keyboards/doio/kb3x/info.json

```diff
@@ -0,0 +1,128 @@
+	        "solid_reactive": true,
+            "solid_reactive_wide": true,
+            "solid_reactive_multiwide": true,
+            "solid_reactive_cross": true,
+            "solid_reactive_multicross": true,
+            "solid_reactive_nexus": true,
+            "solid_reactive_multinexus": true,
+            "splash": true,
+            "multisplash": true,
+            "solid_splash": true,
+            "solid_multisplash": true
+        },
+        "driver": "ws2812"
+        "layout": [
+            {"matrix": [3, 0], "x": 0, "y": 64, "flags": 4},
+            {"matrix": [3, 1], "x": 45, "y": 64, "flags": 4},
+            {"matrix": [3, 2], "x": 90, "y": 64, "flags": 4},
+            {"matrix": [3, 3], "x": 135, "y": 64, "flags": 4},
+            {"matrix": [3, 4], "x": 180, "y": 64, "flags": 4},
+            {"matrix": [3, 5], "x": 224, "y": 64, "flags": 4}
+        ],
+        "max_brightness": 200
+    },
+    "matrix_pins": {
+        "cols": ["B14", "B13", "B12", "B0", "A7", "A6"],
+        "rows": ["B3", "B4", "B9", "B8"]
+    },
+    "diode_direction": "COL2ROW",
+    "dynamic_keymap": {
+        "layer_count": 6
+    },
+    "encoder": {
+        "rotary": [
+            {"pin_a": "B6", "pin_b": "B5", "resolution": 2},           
+            {"pin_a": "A2", "pin_b": "A1", "resolution": 2},
+            {"pin_a": "A4", "pin_b": "A3", "resolution": 2}
+        ]
+    },
+    "ws2812": {
+        "pin": "A10"
+    },
+    "processor": "STM32F103",
+    "bootloader": "stm32duino",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+        "encoder": true
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                { "matrix": [3, 0], "x": 0, "y": 0 },
+                { "matrix": [3, 1], "x": 1, "y": 0 },
+                { "matrix": [3, 2], "x": 2, "y": 0 },
+                { "matrix": [3, 3], "x": 7, "y": 0 },
+                { "matrix": [3, 4], "x": 8, "y": 0 },
+                { "matrix": [3, 5], "x": 9, "y": 0 },
+
+                { "matrix": [0, 0], "x": 1, "y": 2 },
+                { "matrix": [0, 1], "x": 2, "y": 3 },
+                { "matrix": [0, 2], "x": 1, "y": 4 },
+                { "matrix": [0, 3], "x": 0, "y": 3 },
+                { "matrix": [0, 4], "x": 1, "y": 3 },
+
+                { "matrix": [1, 0], "x": 6, "y": 2 },
+                { "matrix": [1, 1], "x": 7, "y": 3 },
+                { "matrix": [1, 2], "x": 6, "y": 4 },
+                { "matrix": [1, 3], "x": 5, "y": 3 },
+                { "matrix": [1, 4], "x": 6, "y": 3 },
+
+                { "matrix": [2, 0], "x": 11, "y": 2 },
+                { "matrix": [2, 1], "x": 12, "y": 3 },
+                { "matrix": [2, 2], "x": 11, "y": 4 },
+                { "matrix": [2, 3], "x": 10, "y": 3 },
+                { "matrix": [2, 4], "x": 11, "y": 3 }
+
```

Comment:
`LAYOUT` sequence matches keymap(s) sequence:
```suggestion
                { "matrix": [3, 0], "x": 0, "y": 0 },
                { "matrix": [3, 1], "x": 1, "y": 0 },
                { "matrix": [3, 2], "x": 2, "y": 0 },

                { "matrix": [3, 3], "x": 7, "y": 0 },
                { "matrix": [3, 4], "x": 8, "y": 0 },
                { "matrix": [3, 5], "x": 9, "y": 0 },

                { "matrix": [0, 0], "x": 1, "y": 2 },

                { "matrix": [1, 0], "x": 6, "y": 2 },

                { "matrix": [2, 0], "x": 11, "y": 2 },

                { "matrix": [0, 3], "x": 0, "y": 3 },
                { "matrix": [0, 4], "x": 1, "y": 3 },
                { "matrix": [0, 1], "x": 2, "y": 3 },
                
                { "matrix": [1, 3], "x": 5, "y": 3 },
                { "matrix": [1, 4], "x": 6, "y": 3 },
                { "matrix": [1, 1], "x": 7, "y": 3 },

                { "matrix": [2, 3], "x": 10, "y": 3 },
                { "matrix": [2, 4], "x": 11, "y": 3 },
                { "matrix": [2, 1], "x": 12, "y": 3 },

                { "matrix": [0, 2], "x": 1, "y": 4 },

                { "matrix": [1, 2], "x": 6, "y": 4 },
                
                { "matrix": [2, 2], "x": 11, "y": 4 }
```

Reply:
@dunk2k Thank you for your suggestion.